### ISSUE TITLE
BUG: exampleSite does not load main.min.css.

### ISSUE THREAD

**Issue by koki-nakamura22** [pacollins/hugo-future-imperfect-slim#93]
# Bug Report
## Describe the bug
When I start Hugo server using exampleSite, the following error happend. And does not load main.min.css.
```
Failed to find a valid digest in the 'integrity' attribute for resource 'http://localhost:1313/css/main.min.1b31ef55024da14996b25283db21d8b249bf405d2c3c4da88ea3f262280698cf.css' with computed SHA-256 integrity 'xMSqFvcG6mqcJ35Dzx2mGMTSDUgMR5z4njbozDQf4rM='. The resource has been blocked.
```

## To Reproduce

1. Create new site.
2. Download hugo-future-imperfect-slim to theme directory.
3. Execute the following command in root directory.
```winbatch
xcopy /s /e .\themes\hugo-future-imperfect-slim\exampleSite .
```
4. Execute the following command in root directory.
```winbatch
hugo server
```

## Expected behavior
Load main.min.css.

## Screenshots
![Capture](https://user-images.githubusercontent.com/45049299/64906290-055aa100-d728-11e9-8ef6-26449aa454fb.PNG)

## Environment

 - **Hugo Version:** [0.57.2]
 - **Browser and Version:** [Vivaldi 2.7.1628.33, Google Chrome  76.0.3809.132]
 - **Platform:** [Desktop]
 - **OS:** [Windows 10 (1903)]

**Comment by pacollins** [comment]
@VincentTam This shouldn't be due to not having the extended version, is it?

**Comment by VincentTam** [comment]
> @VincentTam This shouldn't be due to not having the extended version, is it?

@pacollins I think yes, due to this Hugo FAQ:
https://gohugo.io/troubleshooting/faq/#i-get-tocss-this-feature-is-not-available-in-your-current-hugo-version.

We now track the SCSS code, _not_ the compiled CSS code, which has to be compiled with the extended version of Hugo.

I've tried the above steps on Ubuntu (`xcopy /s /e` adapted to `cp -r`) with Hugo extended version (installed with snap), and I _can't_ reproduce this error.

![Screenshot from 2019-09-14 16-03-56](https://user-images.githubusercontent.com/5748535/64909193-47281f00-d709-11e9-9fc0-0135069e04ee.png)

:information_source: I've got a _different_ hash in the path name for the main CSS.
